acceptance_tests/features/pages/external_conversation.py

```diff
@@ -0,0 +1,60 @@
+from acceptance_tests.features.steps.authentication import signed_in_respondent
+from config import Config
+
+from acceptance_tests import browser
+from acceptance_tests.features.pages import create_message_external
+from controllers import messages_controller
+
+
+def go_to():
+    browser.visit(f"{Config.FRONTSTAGE_SERVICE}/secure-message/threads")
+
+
+def go_to_surveys_todo():
+    browser.visit(f"{Config.FRONTSTAGE_SERVICE}/surveys/todo")
+
+
+def select_to_create_message():
+    browser.find_by_id('create-message-link-1').click()
+
+
+def get_page_title():
+    return browser.title
+
+
+def get_datetime():
+    return 'Today'
+
+
+def get_summary_length():
+    return len(browser.find_by_id('message-summary-1'))
+
+
+def get_endswith_ellipsis():
```

Comment:
This should be either the test 
`return external_conversation.get_endswith_ellipsis()[-3:] == '...'` returning True/False. 
Or this should be renamed to something like 'get_subject_text'

Reply:
This is for making sure that the message body gets truncated and the words at the end are cut off and we get the '...' Not quite sure what you're asking?

Reply:
The function name is get_endswith_ellipsis.  At a glance I would expect this to return a true/false value, as it checks some text ending with ... .  This isn't what this does, this function simply gets some text and returns it, which isn't what the function name describes.
My suggestions are either change the name to make the description of it more accurate (probably the easier option).  Or do the comparison in this function and return a true/false value to assert inside the step.
Does that make sense?